lib/registries/addon/overview/template.hbs

```diff
@@ -62,9 +62,21 @@
                 <layout.leftNavOld as |leftNav|>
                     <leftNav.link data-analytics-name='Overview' @route='registries.overview.index'
                         @models={{array this.registration.id}} @icon='home' @label={{t 'registries.overview.title'}} />
-                    <leftNav.link data-analytics-name='Files' @route='registries.overview.files'
-                        @models={{array this.registration.id}} @icon='file-alt'
-                        @label={{t 'registries.overview.files.title'}} />
+                    {{#if this.features.registrationFilesPage}}
```

Comment:
I think this isn't working, and that's the reason that mocking the service wasn't working. I tested this locally with the flag active, and instead of transitioning to the files route, it looked like it actually went to localhost/<guid>/files and reloaded the app.